README.md

```diff
@@ -1,6 +1,20 @@
-# rdvue cli [![npm version](https://badge.fury.io/js/rdvue.svg)](https://badge.fury.io/js/rdvue)
+<div align="center">
+  <br/>
+  <a href="https://www.realdecoy.com/jamaica/" title="REALDECOY">
+    <img width=710px src="https://www.realdecoy.com/wp-content/uploads/2019/02/Realdecoy-logo-transparent.png" alt="rd logo">
+  </a>
+  <br/>
```

Comment:
You can also remove all these line breaks. I'm not sure we need such a large separation between the logo and rdvue title.